scripts/mppc_to_multiprobe.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/python
+# -*- encoding: utf-8 -*-
+"""
+Created on 22 Aug 2019
+
+@author: Sabrina Rossberger, Delmic
+
+This script provides a command line interface for aligning of the MPPC detector to multiprobe.
+Preparation:
+    On the SEM Control PC:
+        Set HFW to 100 µm (a few times the size of a single field)
+        Set beam shift to zero
+        Select single-beam mode; use beamlet D4.
+        TODO only mppc cell D4 should be active
+        Set the scan mode to ‘External’
+    For scan control:
+        Set galvo mirror deflection to zero; X = Y = 0.
+    Configure the camera to acquire a single field image.  TODO @Job/Thera? single mppc cell scan over time
+
+Scan with a single beamlet the full mppc chip. The effective pixel size (equal to ebeam step size) for the scan
+should be smaller than the physical dimensions of the mppc cell (about 3mm). Only one mppc cell should be active.
+For now all mppc cells are active. Need to process the received full image and cut out the relevant cell.
+When the beamlet hits the active mppc cell, the intensity received for different beamlet
+positions will vary depending on where the mppc cell was hit by the beam. If the beamlet is ideally centered on
+the mppc cell, the highest intensity found during the scan will be in the center of the mppc cell image.
+If not centered, the position of the peak intensity can be calculated and its distance to the center of the mppc
+image. This corresponds to the offset, which can be corrected by adjusting the position of the galvo mirrors.
+"""
+from __future__ import division
+
+import argparse
+import logging
+import math
+import numpy
+
+from odemis import model
+from odemis.dataio import hdf5
+from odemis.util import almost_equal
+from scipy.signal import correlate
+import sys
+from builtins import input
+
+from odemis.acq.align.spot import FindSpot
+
+
+def get_mppc_rotation(image):
+    """
+    Calculates the rotation of the MPPC detector in space. MPPC detector should be horizontally positioned.
+    The scanning is performed with one beamlet over a field of view larger than the effective size of the single
+    MPPC cells.
+    TODO For now all cells are active. => Need to cut the respective mppc cell images.
+    TODO: specify if all 4 mppc cells can be activated during the same scan or if 4 images are needed.
+        If possible to activate all 4 cells in same scan, need multi spot detection.
+    Note: Image needs to be acquired on homogeneous non-sample area.
+    :param image: (ndarray) The full mppc detector image to run the calibration for.
+    :returns: (tuple of 2 float) The difference of the corner cells A8 to A1 in x and the difference
+                of H1 to A1 in y.
+    """
+    # Get the 4 corner mppc cell images: A1, H1, A8, H8
+    # Size of one mppc cell image is (800, 800) TODO @Job correct?
+    # Shape of full mppc is (8, 8)
+    image_A1 = image[0:800, 0:800]
```

Comment:
To make it independent of the amount of pixels in a cell, I would change this to something like this:

```
cell_size_x = image.shape[0] / 8
cell_size_y = image.shape[1] / 8
image_A1 = image[0:cell_size_x, 0:cell_size_y]
...
...
...
```

This works because you know that there are only 8 cells